src/email/send.html

```diff
@@ -1,35 +1,35 @@
-<form ng-submit="sendEmail()" name="emailForm" >
+<form id="email.send.form" ng-submit="sendEmail()" name="emailForm">
     <legend class="frame-title">{{'email.compose.title' | translate}}</legend>
     <fieldset>
         <div class="form-group">
-            <label for="fromAddress">{{'email.from.address' | translate}}</label>
-            <input type="email" ng-model="mail.fromAddress" id="fromAddress" name="fromAddress" aria-describedby="emailForm-fromAddress-errors" required />
-            <ul id="emailForm-fromAddress-errors" class="form-hint">
-                <li ng-if="emailForm.$submitted && emailForm.fromAddress.$error.required ">{{'email.field.required' | translate }} {{'email.from.address' | translate}}</li>
-                <li ng-if="emailForm.$submitted && emailForm.fromAddress.$error.email ">{{'email.field.notValid' | translate}} {{'email.from.address' | translate}}</li>
+            <label for="email.send.from-address">{{'email.from.address' | translate}}</label>
+            <input id="email.send.from-address" type="email" ng-model="mail.fromAddress" name="fromAddress" aria-describedby="emailForm-fromAddress-errors" required />
```

Comment:
@jhopen -- the ID on the `<ul>` element is for accessibility, and is referenced by the aria-describedby property // please make sure this matches...

Can we change the format for the error message IDs? something like "email.send.from-address.errors.required"

The `<li>` element ids are missing an "s"